packages/webapp/components/footer/FooterNavBarTabs.tsx

```diff
@@ -82,12 +88,45 @@ const Tab = ({ tab, isActive }: TabProps) => {
   );
 };
 
+const Notifications = ({ active }: { active: boolean }): JSX.Element => {
+  const { unreadCount } = useNotificationContext();
+
+  return (
+    <div className="relative">
+      <BellIcon secondary={active} size={IconSize.Medium} />
+      {!!unreadCount && (
+        <Bubble
+          className={classNames('-right-1.5 -top-1.5 cursor-pointer px-1')}
+        >
+          {getUnreadText(unreadCount)}
+        </Bubble>
+      )}
+    </div>
+  );
+};
+
 export function FooterNavBarTabs({
   activeTab,
 }: FooterNavBarContainerProps): ReactElement {
+  const notificationsNavBar = useFeature(feature.notificationsNavBar);
+
+  const tabs = useMemo(() => {
+    if (notificationsNavBar) {
+      mobileUxTabs.pop();
+      mobileUxTabs.splice(-1, 0, {
+        requiresLogin: true,
+        path: '/notifications',
+        title: 'Notifications',
+        icon: (active: boolean) => <Notifications active={active} />,
+      });
+    }
+
+    return mobileUxTabs;
+  }, [notificationsNavBar]);
```

Comment:
Had to memoize it, unless you'd end up with lots of notification buttons
![image](https://github.com/dailydotdev/apps/assets/1681525/33181583-7d53-4224-9b1d-c6a4eb06d47b)